src/Apps/Order/Routes/Shipping2/Components/FulfillmentDetails.tsx

```diff
@@ -221,6 +221,7 @@ export const FulfillmentDetails: FC<FulfillmentDetailsProps> = ({
           saveFulfillmentDetailsResult.data.requiresArtsyShippingToDestination
         ) {
           shippingContext.actions.setStage("shipping_quotes")
+          // TODO: jumpTo(shipping quotes?)
```

Comment:
Caught this while going over the code and ticketed in [EMI-2017](https://artsyproduct.atlassian.net/browse/EMI-2071)